tests/replication/slave/storage/test_receipts.py

```diff
@@ -1,265 +0,0 @@
-# Copyright 2016 OpenMarket Ltd
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-from synapse.api.constants import ReceiptTypes
-from synapse.replication.slave.storage.receipts import SlavedReceiptsStore
-from synapse.types import UserID, create_requester
-
-from tests.test_utils.event_injection import create_event
-
-from ._base import BaseSlavedStoreTestCase
-
-OTHER_USER_ID = "@other:test"
-OUR_USER_ID = "@our:test"
-
-
-class SlavedReceiptTestCase(BaseSlavedStoreTestCase):
```

Comment:
I'd be curious if removing the tests lowers coverage at all -- I don't see any other tests calling at least `get_receipts_for_user_with_orderings` which makes me suspicious that this is actually just testing the receipts code and has nothing to do with replication.

Reply:
Good spot, have re-created the test in https://github.com/matrix-org/synapse/pull/13349/commits/3ae8c7ea6934ec7716fe2e114a5d78ea2ae28407.